### PR TITLE
frontend: created bitsurance terms component

### PR DESCRIPTION
**Pull request by shonsirsha**
also moved all terms component (`pocket-terms`, `moonpay-terms`, and `bitsurance-terms`) into a dedicated 'terms' directory.

[Preview (bitsurance-terms)]: 

eqwdas

### CODE REVIEW

File: frontends/web/src/components/terms/bitsurance-terms.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import style from './terms.module.css';
+
+type TProps = {
+  onAgreedTerms: () => void;
+}
+
+export const BitsuranceTerms = ({ onAgreedTerms }: TProps) => {
+  const { t } = useTranslation();
+  const handleSkipDisclaimer = (e: ChangeEvent<HTMLInputElement>) => {
+    setConfig({ frontend: { skipPocketDisclaimer: e.target.checked } });
```
Comment (Beerosagos) [inline frontends/web/src/components/terms/bitsurance-terms.tsx:31]: `skipPocketDiscalimer` should be updated :)

File: frontends/web/src/components/terms/bitsurance-terms.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <div className={style.disclaimerContainer}>
+      <div className={style.disclaimer}>
+        <p>{t('bitsurance.terms.text1')}</p>
+        <p>{t('bitsurance.terms.text2')}</p>
+        <p>{t('bitsurance.terms.text3')}</p>
+        <p>{t('bitsurance.terms.text4')}</p>
+        <p>
+          {t('bitsurance.terms.text5')}
+          {' '}
+          <A href="https://bitsurance.eu/en/dataprotection">
```
Comment (Beerosagos) [inline frontends/web/src/components/terms/bitsurance-terms.tsx:44]: Please use `www.bitsurance.eu` as it is the whitelisted domain in the backend

**Review by Beerosagos — CHANGES_REQUESTED** [review]
Found a couple issues, but also the page is not checking the config set to skip the disclaimer on load. Could you add it please?

File: frontends/web/src/components/terms/bitsurance-terms.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    <div className={style.disclaimerContainer}>
+      <div className={style.disclaimer}>
+        <p>{t('bitsurance.terms.text1')}</p>
+        <p>{t('bitsurance.terms.text2')}</p>
+        <p>{t('bitsurance.terms.text3')}</p>
+        <p>{t('bitsurance.terms.text4')}</p>
+        <p>
+          {t('bitsurance.terms.text5')}
+          {' '}
+          <A href="https://www.bitsurance.eu/en/dataprotection">
```
Comment (thisconnect) [inline frontends/web/src/components/terms/bitsurance-terms.tsx:44]: Link to https://www.bitsurance.eu/datenschutz/ would be nice to have too.

**Review by thisconnect — CHANGES_REQUESTED** [review]

File: frontends/web/src/components/terms/bitsurance-terms.tsx (hunk 3 above)
Comment (Beerosagos) [reply frontends/web/src/components/terms/bitsurance-terms.tsx:44]: Good point, actually! Thanks

Comment (thisconnect) [reply frontends/web/src/components/terms/bitsurance-terms.tsx]: We used to put it in locize but it got messy quickly and afaik we started putting having links in JSX directly with language switch. I guess German-first is ok in this case (their english page is not ready yet).

**Review by thisconnect — APPROVED** [review]
LGTM

**Pull request merged**